snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -135,9 +137,21 @@ def initialize_replica(self, source_adapter_name: str, override_image: str = Non
             override_image: the name of incremental image to initialize,
                 if specified will override default image
         """
-        if override_image:
-            self.__class__.DOCKER_IMAGE = override_image
         try:
+            if override_image:
+                shdocker = SnowShuDocker()
+                images = shdocker.client.images.list(name=override_image)
+                if len(images) > 0:
+                    image_commands = []
+                    for item in images[0].history():
+                        if ("postgres" in item["CreatedBy"]) or ("PGDATA" in item["CreatedBy"]):
+                            image_commands.append(item["CreatedBy"])
+                    if len(image_commands) > 0:
+                        self.__class__.DOCKER_IMAGE = override_image
+                    else:
+                        logger.error(f"The override image is not a Postgres image: {override_image}")
```

Comment:
This `else` block also should raise an error. If there are problems with the image passed to `--incremental`, snowshu should probably abort instead of falling back to a full replica build.